### ISSUE TITLE
2.1 syntax support

### ISSUE THREAD

**Issue by t1bb4r** [leafgarland/typescript-vim#106]
I would love to see this vim plugin updated for the 2.1 syntax enhancements.

I've started using the new typescript 2.1,constructs like Partial and noticed the plugin doesn't recognise the word. The official typescript compiler has no problem compiling the code.
![partial](https://cloud.githubusercontent.com/assets/1861255/22879360/55a26780-f1e6-11e6-9d90-146b81e70fe2.png)

Even if there is just a workaround to suppress the error I would be happy.

**Comment by leafgarland** [comment]
You're right that this syntax plugin does not highlight things like `Partial<ITransaction>` but the compile error is not because of this plugin. I would guess that it is picking up an older `tsc` or perhaps there are specific flags needed to enable typescript to compile them?

**Comment by leafgarland** [comment]
Re-open because we should highlight those correctly.

**Comment by t1bb4r** [comment]
Forgive my ignorance, I have zero experience with vim plugins. I'm an idiot after all, my PATH wasn't set up correctly so the plugin was using my global typescript (npm -g) which was an older version than my project specific version.

Thanks for the quicky reponse.
